omegaconf/listconfig.py

```diff
@@ -539,7 +539,12 @@ def __iadd__(self, other: Iterable[Any]) -> "ListConfig":
         return self
 
     def __contains__(self, item: Any) -> bool:
-        for x in iter(self):
+        if self._is_none() or self._is_missing():
+            return False
```

Comment:
Looking at the previous code, I think it was crashing in this scenario. I believe this would be safer, no?

Reply:
Yes, you are right.
I actually didn't realize that. in any case, no one complained about the previous behavior (probably because you are not supposed to be doing it anyway).

Reply:
Oh, by "this" do you mean return False or raising an exception?

Reply:
I meant that crashing seems safer to me (and noone should complain about a crash when trying to iterate on None / missing)

Reply:
yeah, I agree.
fixed it back.